hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/rollout/rolloutgroup/RolloutGroupGridLayout.java

```diff
@@ -85,6 +104,11 @@ private Optional<Long> getMasterEntityId() {
         return Optional.ofNullable(rolloutGroupListGrid.getMasterEntitySupport().getMasterId());
     }
 
+    @Override
+    public void onViewEnter() {
+        rolloutGroupListGrid.alignWithConsentFlowState();
+    }
+
```

Comment:
Every time we enter the rollout view, the column existence is re-evaluated based on the tenant configuration and column presence in the grid. Then why do we need a specific tenantConfigChangelistener? Or am I missing something here?

Reply:
The change listener will trigger the re-evaluation if the config was changed by a different UI operator.